app/src/main/java/com/example/music_app/data/remote/Contract/MainContract.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.music_app.data.remote.Contract
+
+import android.content.Context
+import com.example.music_app.Model.Entity.Entity.Song
+import com.example.music_app.Service.MusicPlayService
+
+
+interface MainContract {
+    interface View {
+        fun showList(songs: ArrayList<Song>)
```

Comment:
```suggestion
        fun displaySongs(songs: List<Song>)
```
tên method cần có ý nghĩa hơn